packages/core/src/utils/classy/classy.ts

```diff
@@ -19,73 +18,39 @@ type Template = [TemplateStringsArray, ...ClassNames[]];
  * // 'foo-1g4k53 global-1 global-2 className disabled'
  */
 export const classy = (...classNames: ClassNames[]) =>
-  classNames
-    .flat(10)
-    .flatMap((className) =>
-      className instanceof Object ? getTruthyKeys(className) : className
-    )
-    .filter(Boolean)
-    .join(' ');
+  classNames.flat(10).flatMap(truthyNames).join(' ');
 
 /**
  * Prefixes component names with a predetermined identifier.
```

Comment:
I suppose it should be singular?

```suggestion
 * Prefixes component name with a predetermined identifier.
```

Reply:
Well theoretically it could be multiple, e.g. `c('card', 'accordion')` 🤷🏻

Reply:
Ah, yes, you're right! Ignore this...